vue/src/vue-elements/main-page-panel.vue

```diff
@@ -1,88 +1,182 @@
 <template>
-    <div>
-        <div class="columns panel-header">
-            <div class="column header-logo vertical-align" v-if="is_preloading_over > 0">
-                <div>
-                    <img :src="plugin_logo" class="plugin-logo avatar avatar-lg"/>
-                    <h1 class="plugin-title d-inline-block">Revive Old Posts</h1><span class="powered d-inline-block"> {{labels.by}} <a
-                        href="https://revive.social" target="_blank"><b>Revive.Social</b></a> <a href="https://www.youtube.com/playlist?list=PLAsG7vAH40Q512C8d_0lBpVZusUQqUxuH" target="_blank" id="rop_vid_tuts"><span>START HERE <i class="fa fa-play-circle" aria-hidden="true"></i></span></a></span>
-                    
-                </div>
-            </div>
-            <toast/>
-            <div v-if=" is_rest_api_error " class="toast toast-error rop-api-not-available" v-html="labels.api_not_available">
-            </div>
-            <div v-if=" is_fb_domain_notice " class="toast toast-primary">
-                <button class="btn btn-clear float-right" @click="close_fb_domain_notice()"></button>
-                <div v-html="labels.rop_facebook_domain_toast"></div>
-            </div>
-            <div class="sidebar sidebar-top card rop-container-start">
-
-                <!-- Next post count down -->
-                <countdown :current_time="current_time"/>
-                <!--  -->
-
-                <button class="btn btn-sm" :class="btn_class"
-                        :data-tooltip="labels.active_account_warning"
-                        @click="togglePosting()" :disabled="!haveAccountsActive">
-                    <i class="fa fa-play" v-if="!is_loading && !start_status"></i>
-                    <i class="fa fa-stop" v-else-if="!is_loading && start_status"></i>
-                    <i class="fa fa-spinner fa-spin" v-else></i>
-                    {{( start_status ? labels.stop : labels.start )}} {{labels.sharing}}
-                </button>
-            </div>
+  <div>
+    <div class="columns panel-header">
+      <div
+        v-if="is_preloading_over > 0"
+        class="column header-logo vertical-align"
+      >
+        <div>
+          <img
+            :src="plugin_logo"
+            class="plugin-logo avatar avatar-lg"
+          >
+          <h1 class="plugin-title d-inline-block">
+            Revive Old Posts
+          </h1><span class="powered d-inline-block"> {{ labels.by }} <a
+            href="https://revive.social"
+            target="_blank"
+          ><b>Revive.Social</b></a> <a
+            id="rop_vid_tuts"
+            href="https://www.youtube.com/playlist?list=PLAsG7vAH40Q512C8d_0lBpVZusUQqUxuH"
+            target="_blank"
+          ><span>START HERE <i
+            class="fa fa-play-circle"
+            aria-hidden="true"
+          /></span></a></span>
```

Comment:
NIT: The formatting here does not look right.

Reply:
Thanks. Also, I switched to using the Eslint version from `wp-scripts`.